_example/meetings/delete_meeting.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/zoom-lib-golang/zoom-lib-golang"
+)
+
+func main() {
+	var (
+		apiKey    = os.Getenv("ZOOM_API_KEY")
+		apiSecret = os.Getenv("ZOOM_API_SECRET")
+		meetingID = os.Getenv("MEETING_ID")
+	)
+
+	zoom.APIKey = apiKey
+	zoom.APISecret = apiSecret
+
+	id, err := strconv.Atoi(meetingID)
+
+	if err != nil {
+		log.Printf("Error: %+v\n\n", err)
+	}
+
+	deleteMeetingOpt := zoom.DeleteMeetingOptions{
+		MeetingID: id,
+	}
+
+	err = zoom.DeleteMeeting(deleteMeetingOpt)
+
+	if err != nil {
```

Comment:
Please prefer abbreviated syntax here:

```suggestion
	if err = zoom.DeleteMeeting(deleteMeetingOpt); err != nil {
```